### PR TITLE
Modify the fileset template to exclude files by fileset

### PR DESCRIPTION
**Pull request by jeremyblot**
Modify the fileset template to exclude files by fileset

### CODE REVIEW

File: templates/bareos-dir/fileset/fileset.conf.j2 [hunk 1]
```diff
@@ -27,5 +27,10 @@ FileSet {
 {% for file in bareos_excludes %}
     File = {{ file }}
 {% endfor %}
+{% if item.exclude is defined %}
```
Comment (pbouchez) [inline templates/bareos-dir/fileset/fileset.conf.j2]: `{% if item.exclude_file is defined %}` ?

**Review by btravouillon — COMMENTED** [review]
Could you document this in the README?

**Review by ahmam — DISMISSED** [review]

File: README.md [hunk 2]
```diff
@@ -42,6 +42,8 @@ bareos_filesets:
   - name: FilesetFoo
     description: "Backup Foo"
     include_file: /home/foo
+    exclude_file:
+      - /home/foo
```
Comment (btravouillon) [inline README.md]: Can you make the example consistant so the `include_file` and `exclude_file` don't use the same path?

```suggestion
      - /home/foo/bar
```

Comment (jeremyblot) [reply README.md]: I understand, that's true !

File: README.md [hunk 3]
```diff
@@ -42,6 +42,8 @@ bareos_filesets:
   - name: FilesetFoo
     description: "Backup Foo"
     include_file: /home/foo
+    exclude_file:
+      - /home/bar
```
Comment (btravouillon) [inline README.md]:
```suggestion
      - /home/foo/bar
```

**Review by btravouillon — APPROVED** [review]
Looks good to me. 👍

**Review by pbouchez — APPROVED** [review]

**Pull request merged**